### PR TITLE
Clarify timeRemaining() algorithm.

### PR DESCRIPTION
**Pull request by rmcilroy**
Addresses #77

### CODE REVIEW

**Comment by rmcilroy** [comment]
Yoav, PTAL a look, thanks.

File: index.html [hunk 1]
```diff
@@ -453,12 +453,11 @@ <h2>The <code><dfn>IdleDeadline</dfn></code> interface</h2>
       milliseconds of the deadline. This MUST be populated when the
       <a>IdleDeadline</a> is created.</p>
       <p>When the <code><dfn>timeRemaining</dfn>()</code> method is invoked on
-      an <a>IdleDeadline</a>
-      object it MUST return the duration, as a <a>DOMHighResTimeStamp</a>,
-      between the current time and the <a>time</a> associated with the
-      <a>IdleDeadline</a> object. The value SHOULD be accurate to 5 microseconds
-      - see "Privacy and Security" section of [[!HR-TIME]]. This value is
-      calculated by performing the following steps:</p>
+      an <a>IdleDeadline</a> object it MUST return the remain duration before
```
Comment (yoavweiss) [inline index.html]: s/remain/remaining/ ?

File: index.html [hunk 2]
```diff
@@ -453,12 +453,11 @@ <h2>The <code><dfn>IdleDeadline</dfn></code> interface</h2>
...
+      the deadline expires as a <a>DOMHighResTimeStamp</a>, which SHOULD be
+      accurate to 5 microseconds - see "Privacy and Security" section of
```
Comment (yoavweiss) [inline index.html]: I believe the 5 microsecond language is not longer true in a post-Spectre world.

File: index.html [hunk 3]
```diff
@@ -453,12 +453,11 @@ <h2>The <code><dfn>IdleDeadline</dfn></code> interface</h2>
...
+      [[!HR-TIME]]. This value is calculated by performing the following
```
Comment (yoavweiss) [inline index.html]: Nit: no need for "!" in the HR-TIME link

**Review by yoavweiss — COMMENTED** [review]
A few comments. I'm also not 100% clear on the issue @szager-chromium had with the current language, so he might be in a better position to say that it is actually resolved here.

**Comment by szager-chromium** [comment]
lgtm

File: index.html (hunk 2 above)
Comment (rmcilroy) [reply index.html]: Updated to "enough to allow measurement while preventing timing attack" based on hr-time text.

File: index.html (hunk 3 above)
Comment (rmcilroy) [reply index.html]: Done

File: index.html (hunk 1 above)
Comment (rmcilroy) [reply index.html]: Done

File: index.html (hunk 2 above)
Comment (yoavweiss) [reply index.html]: :+1:

**Review by yoavweiss — APPROVED** [review]

**Pull request merged**